CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
+
+### Release Notes
+* Features  
+  * Kubernetes 1.31 and 1.32 are now designated as [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) versions.  
+  * Kubernetes 1.33 is available in Preview. A full matrix of supported add-ons and components is published at the [AKS versions page](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-1330).
+  * Network-isolated clusters are now Generally Available, including GA support for CNIv1 → Cilium and Cilium → Overlay upgrade paths.
+
+* Bug Fixes  
+  * Retained `kube-proxy` on add-ons v2 during Cilium migrations to prevent transient service-connectivity loss.  
+  * Updated the Windows GPU device-plugin to `v0.0.19`, mitigating CVE-2025-22871.  
+  * Switched the static-egress-gateway init container to the pre-built image `v0.0.20`, eliminating failures in environments without Internet access.  
+  * Re-enabled the `ExtensionManager` on Overlay, unblocking Flux extension deployments.  
+  * Overlay manager now ignores maintenance windows that aren’t weekly.  
+  * Fixed failures triggered by duplicate tag keys that differed only by character case.
+  * Defaults variable 'ipFamilies' to be set to IPv4 if not defined. Fixes an issue in some older clusters which blocks Karpenter if variable 'ipFamilies' is set to null.
+
+* Behavior Changes  
+  * In-place upgrades are now supported when the cluster is using [Azure CNI Overlay (preview)](https://learn.microsoft.com/azure/aks/azure-cni-overlay).
+  * Overlay manager disregards non-weekly maintenance windows (see Bug Fixes).  
+  * Duplicate tag keys that differ only by case are now rejected at creation time.
+
+* Component Updates  
+  * CRDs `1.4.0-aks`  
+    * Added zone-label support and image-status fields.  
+    * Introduced `v1beta1` API for AKSNodeClass; default CRs updated accordingly.  
+    * Deployment flow now consumes `vnetGUID` from Secret when present.  
+    * Clean-up of obsolete resource versions/labels; broader ConfigMap read permissions for maintenance-window detection.  
+  * Windows node images  
+    * Server 2019 Gen1 – `17763.7240.250416`  
+    * Server 2022 Gen1/Gen2 – `20348.3561.250416`  
+    * Server 23H2 Gen1/Gen2 – `25398.1551.250416`  
+  * AgentBaker versions bumped to `v0.20250422.0`, `v0.20250509.0`, and `v0.20250514.0`.  
```

Comment:
link the release notes ? we are not generating a more human friendly readable release note in github. 

https://github.com/Azure/AgentBaker/releases/tag/v0.20250422.0
https://github.com/Azure/AgentBaker/releases/tag/v0.20250428.0
https://github.com/Azure/AgentBaker/releases/tag/v0.20250509.0
https://github.com/Azure/AgentBaker/releases/tag/v0.20250514.0